torch/_prims/__init__.py

```diff
@@ -2541,6 +2542,18 @@ def _scalar_tensor_aten(
     doc=_scalar_tensor_doc,
 )
 
+_int_tensor_doc = """
+    Wraps a SymInt or a list of SymInts into a Tensor with the specified dtype and device.
+"""
+
+int_tensor = _make_prim(
+    # TODO: handle scalar values
+    schema="int_tensor(SymInt[] data, *, ScalarType? dtype=None, Device? device=None, bool pin_memory=False, bool requires_grad=False) -> Tensor",
+    meta=torch.tensor,
```

Comment:
This meta is too permissive; you need to only accept lists of ints as the input, arbitrary torch.tensor inputs are not OK.

Reply:
that's already enforced by the schema